docs/melange-book/installation.md

```diff
@@ -0,0 +1,59 @@
+# Installation
+
+## Prerequisites
+
+- You are running a Unix system
+  - Windows users are encouraged to install [Windows Subsystem for
+Linux](https://learn.microsoft.com/en-us/windows/wsl/)
+- You have a recent version of, and know how to use:
+  - [Node.js](https://nodejs.org/)
+  - [git](https://git-scm.com/)
+  - [Visual Studio Code](https://code.visualstudio.com/)
+
+## Opam
+
+We need [opam](https://opam.ocaml.org/), the OCaml Package Manager. There are
+many ways to install it depending on your platform, but let's go with the
+simplest method:
+
+    bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+    opam init
+    eval $(opam env)
```

Comment:
You can point out that `opam env` is optional and it does bring all binaries into $PATH, but if you don't like it you can always `opam exec -- xxx`

Reply:
Is there some major downside to `opam env`? If not, I'd prefer not to get  into the weeds here.